lib/mapbox-gl-geocoder.css

```diff
@@ -3,137 +3,174 @@
 .mapboxgl-ctrl-geocoder *,
 .mapboxgl-ctrl-geocoder *:after,
 .mapboxgl-ctrl-geocoder *:before {
-  -webkit-box-sizing:border-box;
-     -moz-box-sizing:border-box;
-          box-sizing:border-box;
-  }
+  box-sizing: border-box;
+}
+
 .mapboxgl-ctrl-geocoder {
-  font:15px/20px 'Helvetica Neue', Arial, Helvetica, sans-serif;
-  position:relative;
-  background-color:white;
-  width:33.3333%;
-  min-width:240px;
-  max-width:360px;
-  z-index:1;
-  border-radius:3px;
-  }
+  font: 15px/20px "Open Sans", "Helvetica Neue", Arial, Helvetica, sans-serif;
```

Comment:
I'm seeing system font in the input box and open sans in the results list:
![Screen Shot 2019-03-19 at 11 39 12 AM](https://user-images.githubusercontent.com/108094/54632607-bdd6af00-4a3b-11e9-8a47-55801e9c4326.png)